assets/css/lesson.scss

```
@@ -225,6 +225,20 @@ a code {
   color: #006cad;
 }
 
+a:link {
+  color: #196EBD;
+}
+
+a:active,
+a:hover {
+  outline: 0;
+  text-decoration: underline;
+}
+
+a:visited {
+  color: #BC19BD;
```

```suggestion
  color: #1AA4DA;
```

Under Color Oracle, the two new link colours actually did not appear very different for people with deuteranopia.